synthclient/manage-exemptions/add-ets

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# This script expects to be called as follows:
+#
+#     cat order.json                       \
+#       | add-ets /path/to/customer/data/  \
+#       > order-with-ets.json
+#
```

Comment:
A bash purist would complain about "redundant use of `cat`", but I think it's clearer to do this and to use both `<` and `>`.